bin/taskcluster/task-definitions.yml

```diff
@@ -0,0 +1,149 @@
+build-lint:
+  payload:
+    maxRunTime: 800
+    image: 'node:6'
```

Comment:
Docker host many image. node:6 is one of the offical image for node, where you can select which version of node you want to be installed.
https://hub.docker.com/_/node/
I chose `node:6` instead of default taskcluster one `rail/python-test-runner` as lint involves node/npm.

See https://tools.taskcluster.net/quickstart/
as a good entrypoint to discover and play with taskcluster.